js/app/controllers/content-product.ng

```diff
@@ -138,7 +138,173 @@ mkdir my-new-bucket</pre>
       the Cloud Tools for PowerShell module via <code>Import-Module GoogleCloud</code>.
     </p>
   </div>
+  
+  <!-- Google Cloud BigQuery -->
+  <div ng-switch-when="google-cloud-bigquery">
+    <p>
+      Google BigQuery is a versatile tool that solves the problem of storing and querying massive 
+      datasets without having to worry about data formats, underlying resources, and other things 
+      that distract you from your analysis.
+    </p>
+    
+    <h2>Dataset</h2>
+    <p>
+      To use BigQuery in a Cloud project, first create a <code>Dataset</code> using the 
+      <cmdlet-ref name="New-BqDataset"></cmdlet-ref> cmdlet. This will take in basic information and 
+      create the resource server-side. Locally, a <code>Dataset</code> <em>reference object</em> is returned. 
+      To get a reference object for an existing dataset, use <cmdlet-ref name="Get-BqDataset"></cmdlet-ref>.
+    </p>
+    <pre>
+# Makes a new dataset with DatasetId "page_views".
+$dataset = New-BqDataset "page_views" -Name "Page Views" -Description "Page views from 2014 onwards"
```

Comment:
We should make `-Description "Page views from 2014 onwards"` go to the next line (using backtick).